src/glfw3Config.cmake.in

```diff
@@ -13,3 +13,4 @@ set_and_check(GLFW3_LIBRARY_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@"
 
 find_library(GLFW3_LIBRARY "@GLFW_LIB_NAME@" HINTS ${GLFW3_LIBRARY_DIR})
 
+set(GLFW_LIBRARIES "@GLFW_LIBRARIES@")
```

Comment:
While we're pluralizing things, set GLFW_INCLUDE_DIRS and GLFW_LIBRARY_DIRS as well.